### PR TITLE
Add project vital statistics and privacy details

### PR DESCRIPTION
**Pull request by BryanQuigley**
Add project vital statistics and privacy details

README
 - Bring in better description
 - Links to prd/stg/marketing sites
 - Add frameworks and history
 - Specify AWS access is required for most tasks on project

Deployment/README
 - Add log details and expirations
 - Add data collected and where it is stored

Disabled tflint check because while it was showing real changes we
should make it, it's not a doc change.

### Notes

Doc change only

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +42,7 @@ $ ./scripts/setup
 
 ### Running
 
-Finally, use the `server` script to build container images, compile frontend assets,
+Finally, use the `server` scripThat site is not hosted by Azavea.t to build container images, compile frontend assets,
```
Comment (ddohler) [inline README.md]:
```suggestion
Finally, use the `server` script to build container images, compile frontend assets,
```

File: README.md [hunk 2]
```diff
@@ -41,7 +57,7 @@ This project uses [`scripts-to-rule-them-all`](https://github.com/azavea/archite
 | Script      | Use                                                        |
 | ----------- | ---------------------------------------------------------- |
 | `bootstrap` | Pull down secrets from S3                                  |
-| `infra`     | Execute Terraform subcommands with remote state management |
+| `infra`     | Execute TerrafoThat site is not hosted by Azavea.rm subcommands with remote state management |
```
Comment (ddohler) [inline README.md]:
```suggestion
| `infra`     | Execute Terraform subcommands with remote state management |
```

File: deployment/README.md [hunk 3]
```diff
@@ -17,6 +19,31 @@ Default output format [None]:
...
+This is tracked for general public usage. Azavea or customer account usage is not included.
+
+- Emails
+- IP addresses (only in logs)
+- Budget for apartment
+- Information that could determine size of family or if person has or is planning to have kids
+- Common places visited. This is user customizable and can include any address.
+- Preferred travel mode (may mirror car ownership)
+
+Most PII is stored in Local Storage, in the RDS database, or both.  Their is no expiration for Local storage or database.
```
Comment (ddohler) [inline deployment/README.md]:
```suggestion
Most PII is stored in Local Storage, in the RDS database, or both.  There is no expiration for Local Storage or database.
```

**Review by ddohler — APPROVED** [review]
Looks great! I had one typo-fixing comment. I also started looking before you asked for review and saw some stray changes that it looks like you've addressed, but now I can't delete those comments because the commits are gone. So apologies if you get notifications about things you've already fixed!

File: README.md (hunk 1 above)
Comment (BryanQuigley) [reply README.md]: Already done :)

File: README.md (hunk 2 above)
Comment (BryanQuigley) [reply README.md]: Already done :)

File: deployment/README.md (hunk 3 above)
Comment (BryanQuigley) [reply deployment/README.md]: added

File: README.md [hunk 4]
```diff
@@ -1,6 +1,22 @@
...
+
+There are two instances hosted based on the code in this repo:
+
+- [Prod](https://app.echosearch.org)
+- [Stg](https://stg.echosearch.org)
+
+## Frameworks and History
+
+- Django backed by Postgres.
+- AWS Lambda and API Gateway for Realtor listings.
```
Comment (ktohalloran) [inline README.md]: Should we also add AWS DynamoDB here?

File: deployment/README.md [hunk 5]
```diff
@@ -17,6 +19,31 @@ Default output format [None]:
...
+- Emails
+- IP addresses (only in logs)
+- Budget for apartment
+- Information that could determine size of family or if person has or is planning to have kids
+- Common places visited. This is user customizable and can include any address.
+- Preferred travel mode (may mirror car ownership)
+
+Most PII is stored in Local Storage, in the RDS database, or both.  There is no expiration for Local Storage or database.
+
+No usernames or passwords are stored for the Django version of this app, it uses email driven passwordless logins with 30 day session timeout.
```
Comment (ktohalloran) [inline deployment/README.md]: Should we specify that login emails expire after an hour?

File: deployment/README.md [hunk 6]
```diff
@@ -17,6 +19,31 @@ Default output format [None]:
...
+- Rollbar (if an error) - 180 day retention
+- AWS Cloudwatch - Django app - 1 month retention
+- AWS Cloudwatch - a lot of older logs that don't expire, and some logs from Realtor and other processes
+- AWS Builtin Logging - From ECS/RDS, using AWS defaults which is 35 days or less.
+- AWS S3 logs - New django style/Cloudfront - stored indefinitely, but not setup correctly
+- AWS S3 logs - old setup - stored indefinitely, can likely be deleted?
+
+## Personally Identifiable Information Collected
+
+This is tracked for general public usage. Azavea or customer account usage is not included.
```
Comment (ktohalloran) [inline deployment/README.md]: I'm not sure I understand what you mean here; it kind of sounds like we give the general public access to these stats, but that's obviously not right. Does this mean we only track general public usage without collecting specific stats about individual usage?

**Review by ktohalloran — APPROVED** [review]
Looks good, this is all really helpful information to have in one place! Just a few questions below, but otherwise 👍

File: README.md (hunk 4 above)
Comment (BryanQuigley) [reply README.md]: Defintiely

File: deployment/README.md (hunk 5 above)
Comment (BryanQuigley) [reply deployment/README.md]: Yes, added.

File: deployment/README.md (hunk 6 above)
Comment (BryanQuigley) [reply deployment/README.md]: Yea, that is not super clear.  I revised to:

"We are tracking this for housing searchers only and not for Azavea, BHA or other management accounts."  

Is that better?

Comment (ktohalloran) [reply deployment/README.md]: Yes, that's much clearer. Thanks!

**Pull request merged**